hw3.js

```diff
@@ -0,0 +1,34 @@
+//Task 1
+let item_1;
+item_1 = 5;
+console.log(item_1)
+let item_2;
+item_2 = 3;
+console.log(item_2);
+let item_3;
+item_3 = item_1 + item_2;
+console.log(item_3);
+let item_4;
+item_4 = "Hello World";
+console.log(item_4);
+console.log(item_3 + item_4)
+console.log(item_3 * item_4);
+let item_5; 
+item_5 = item_3;
+let item_6;
+let item_6_type;
+item_6 = 15;
+item_6_type = typeof(item_6)
+console.log("item_6 == ", item_6, "item_6_type == ", item_6_type);
```

Comment:
`Вывести в консоль тип данных`, так же для задания ниже
```suggestion
console.log("item_6 == ", typeof item_6, "item_6_type == ", typeof item_6_type);
```